docker/zap_common.py

```diff
@@ -272,7 +272,7 @@ def start_zap(port, extra_zap_params):
 def wait_for_zap_start(zap, timeout_in_secs = 600):
     version = None
     if not timeout_in_secs:
-        # if ZAP doesnt start in 10 mins then its probably not going to start
+        # if ZAP doesn't start in 10 mins then its probably not going to start
```

Comment:
I usually don't bother enforcing apostrophes. I think I mostly offered these because of the `mustn't` cases.